lib/omnibus/project.rb

```diff
@@ -1402,6 +1403,15 @@ def built_manifest
       m
     end
 
+    def build_summary
+      summary = {"build" => {}}
+      softwares.each do |s|
+        summary["build"][s.name] = s.build_summary
+      end
+      summary["packaging"] = @package_summary
+      summary
+    end
```

Comment:
🤔 might be me being a ruby noob but I'm not finding where this function is called ?

Reply:
Right there: https://github.com/DataDog/omnibus-ruby/pull/183/files#diff-07a7d7bffaf8905a505d0201c036661c95f08b2cb4d5d86c13f7fe774bcfe9e1R1472
I agree that it's not obvious at all, but it duplicates what's done for the catalog manifest